src/client/components/CompanyLocalHeader/LocalHeaderCompanyRefer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
```suggestion
import React from 'react'
import styled from 'styled-components'
import { FONT_SIZE } from '@govuk-react/constants'

import { GREY_3, GREY_3_LEGACY } from '../../../client/utils/colours'
```

The third-party imports should be grouped together and separated from the internal ones by a line break.